anvil/observables.py

```diff
@@ -14,6 +14,7 @@
 import torch
 from tqdm import tqdm
 
+
```

Comment:
I think we will just resolve the conflict by abandoning the changes to observable

Reply:
in fact all of the formatting changes which I guess were made by `black` we will probbaly abandon here, since otherwise the files are untouched